mei2ly.xsl

```diff
@@ -3853,6 +3855,16 @@
       </xsl:otherwise>
     </xsl:choose>
   </xsl:template>
+  <xsl:template mode="addBeamMarkup" match="*"/>
```

Comment:
Yes, because otherwise we only covered the cases where `key('isBeamStart',...)` and `key('isBeamEnd',...)` apply. In all other cases, the [Built-in template rules](https://www.w3.org/TR/xslt#built-in-rule) would fire if the line was absent. This is a problem e.g. when a `<note>` has lyrics in a `<syl>` element.